lib/order.rb

```diff
@@ -0,0 +1,53 @@
+require 'twilio-ruby'
+
+class Order
+  attr_reader :order, :menu
+
+  def initialize(menu)
+    @order = []
+    @menu = menu
```

Comment:
One thing that can reduce the need for class Order to be dependant on class Menu is to add (menu = Menu.new) within the initialize argument.